SpatialGDK/Documentation/content/directory-structure.md

```diff
@@ -1,22 +1,19 @@
-| `linting/` | Contains the linting scripts for checking the GDK for Unreal.
-| `docs` | Contains the documentation for the GDK for Unreal.
-| `fastbuild` | Contains files related to installing FASTBuild. **Note** these currently only work for Improbable engineers.
-| `packages/` | (Not tracked in git) Contains the dependencies for building the Unreal SDK codegen.
-| `Plugins/SpatialGDK/SpatialGDKEditorToolbar/` | The [SpatialOS GDK for Unreal toolbar]({{urlRoot}}/content/toolbar.md) within the Unreal Editor, from which you can take snapshots, generate interop code, start and stop deployments, and access the SpatialOS Inspector.
-| `schema` | Contains the schema files required for the GDK to interact with SpatialOS.
-| `Scripts/` | Contains a set of helper scripts used to build a GDK worker.
-| `Source/Programs/` | Contains the Unreal SDK code generator.
-| `Source/SpatialGDK/Legacy/` | Contains the source code from the [GitHub-hosted Unreal SDK](https://github.com/spatialos/UnrealSDK/tree/master/Game/Source/SpatialOS).
-| `Source/SpatialGDK/Legacy/Deprecated/` | Contains the forked source code from the [GitHub-hosted Unreal SDK](https://github.com/spatialos/UnrealSDK/tree/master/Game/Source/SpatialOS) that the GDK does not depend on. This is present purely for backwards compatibility.
-| `Source/SpatialGDK/Public/WorkerSdk/` | (Not tracked in git) Contains the [C++ worker SDK](https://docs.improbable.io/reference/latest/cppsdk/introduction) headers. You install these when you run `BuildGDK.bat`
-| `Source/SpatialGDK/Public` | Public source code of the GDK for Unreal module.
-| `Source/SpatialGDK/Private` | Private source code of the GDK for Unreal module.
+| `SpatialGDK/Binaries/ThirdParty/Improbable/` | (Not tracked in git) This directory contains all binaries required for your game project with the GDK for Unreal. These files are generated by `BuildGDK.bat`.
+| `SpatialGDK/Build/core_sdk/` | (Not tracked in git) Contains the [C API worker SDK](https://docs.improbable.io/reference/latest/capi/introduction) depndencies used by the Unreal GDK to serialize data to and from SpatialOS.
+| `SpatialGDK/Build/Programs/` | Contains the source and project files for the executables used when building the GDK.
+| `SpatialGDK/Build/Scripts/` | Contains a the helper scripts that allow you to build either a server or a client worker.
+| `SpatialGDK/Documentation` | Contains the documentation for the GDK for Unreal.
+| `SpatialGDK/Extras/fastbuild` | Contains files related to [FASTBuild](http://www.fastbuild.org/docs/home.html), an open-source build system that is currently only usable by Improbable engineers.
+| `SpatialGDK/Extras/schema` | Contains the [schema files](https://docs.improbable.io/reference/latest/shared/glossary#schema) required for the GDK to interact with SpatialOS.
+| `SpatialGDK/Extras/linting/` | Contains the scripts we use to lint the Unreal GDK.
```

Comment:
"...the GDK."
(Once we have used "GDK for Unreal" in a page, we can then go to "GDK", I think. (Makes it easier to read.)
We have to avoid "Unreal GDK".